public/index.html

```diff
@@ -25,7 +25,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" class="root-container"></div>
```

Comment:
normally, we don't style for this div, it's purely just used to inject the React App to this DOM. Let's add the style on the App Root component